src/lib/checkVersion.ts

```diff
@@ -61,7 +66,7 @@ export class CheckVersion {
         )
         return isNewVersion
       } else {
-        this.core.setOutput('version', version)
+        this.core.setOutput('version', `v${version}`)
```

Comment:
see how this https://github.com/CDECatapult/das-lambda-ui-api-reporting/actions/runs/10270405224/job/28418051584
produced a release/tag with no `v` https://github.com/CDECatapult/das-lambda-ui-api-reporting/releases/tag/0.0.1